ivy/functional/frontends/tensorflow/layers.py

```diff
@@ -0,0 +1,6 @@
+#global
+import ivy
+import ivy.functional.frontends.tensorflow as tf_frontend
+
+def tf_frontend.keras.layers.concatenate(value, axis, name="ivy.functional.frontends.tensorflow.keras.layers.concatenate"):

```

Comment:
Your frontend function should have the same argument as the target function that we want to recreate:
tf.keras.layers.concatenate(
    inputs, axis=-1,
)
the function name should be a simple "concatenate". P.S. I do not know why you add those prefixes when you define function in python.

Reply:
when I pulled the previous pull request I formed the function with a similar name but the assigned person. told me that I should change the name from layers_concatenate in the Tensorflow docs. The function signature of our frontend should match the signature of the original Tensorflow function, so if this function is a frontend implementation of tf.keras.layers.concatenate this function should be callable from ivy.functional.frontends.tensorflow.keras.layers.concatenate

Reply:
Well, you should refer to the tensorflow example here: https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html